### PR TITLE
Bytestrings

### PR DESCRIPTION
**Pull request by Chris-Corea**

### CODE REVIEW

File: formats/cbor/commonTest/src/kotlinx/serialization/cbor/CborReaderTest.kt [hunk 1]
```diff
@@ -36,6 +37,49 @@ class CborReaderTest {
         }
     }
 
+    @Test
+    fun testDecodeByteStrings() {
+        // 12 bytes
+        withDecoder("4C6332383500000000000001AD") {
```
Comment (twyatt) [inline formats/cbor/commonTest/src/kotlinx/serialization/cbor/CborReaderTest.kt:43]: Where did `4C6332383500000000000001AD` come from?

File: formats/cbor/commonTest/src/kotlinx/serialization/cbor/CborReaderTest.kt [hunk 2]
```diff
@@ -36,6 +37,49 @@ class CborReaderTest {
...
+            assertEquals(
+                expected = "6332383500000000000001AD",
+                actual = HexConverter.printHexBinary(nextByteString())
+            )
+        }
+
+        // 30 bytes
+        withDecoder("581E6332383500000000000001AD9E8BAC4209005A478736AD000010011001AD") {
```
Comment (twyatt) [inline formats/cbor/commonTest/src/kotlinx/serialization/cbor/CborReaderTest.kt:51]: This too?
`581E6332383500000000000001AD9E8BAC4209005A478736AD000010011001AD`

**Comment by Chris-Corea** [comment]
@twyatt `4C6332383500000000000001AD` and `581E6332383500000000000001AD9E8BAC4209005A478736AD000010011001AD` were just arbitrary cbor objects that I made up for testing purposes. Using [cbor.me](cbor.me):
```
// both are just plain byte strings
4C6332383500000000000001AD = h'6332383500000000000001AD'
581E6332383500000000000001AD9E8BAC4209005A478736AD000010011001AD = h'6332383500000000000001AD9E8BAC4209005A478736AD000010011001AD'
```

**Comment by twyatt** [comment]
> @twyatt `4C6332383500000000000001AD` and `581E6332383500000000000001AD9E8BAC4209005A478736AD000010011001AD` were just arbitrary cbor objects that I made up for testing purposes. Using [cbor.me](cbor.me):
> 
> ```
> // both are just plain byte strings
> 4C6332383500000000000001AD = h'6332383500000000000001AD'
> 581E6332383500000000000001AD9E8BAC4209005A478736AD000010011001AD = h'6332383500000000000001AD9E8BAC4209005A478736AD000010011001AD'
> ```

Awesome, thanks for clarifying. I was going to change it to more recognizable test data (e.g. `0x01`, `0x02`, ...) and wanted to make sure I wasn't getting rid of data that had specific content for a reason.

**Pull request merged**